docs/registry.md

```diff
@@ -58,7 +64,7 @@ version = 2
     key_file  = "key.pem"
 ```
 
-In the config example shown above, TLS mutual authentication will be used for communications with the registry endpoint located at https://my.custom.registry.
+In the config example shown above, TLS mutual authentication will be used for communications with the registry endpoint located at <https://my.custom.registry>.
```

Comment:
did you want a `[ref](link)`

Reply:
Markdown linter prefers it within `<>`, thats all

Reply:
What do you think?

Reply:
unfortunately the preview code on github here... would not show what that ^ looks like.  After merging.. I see it shows it as a clickable link.  I don't think that was the intent. Maybe `https://my.custom.registry.` Cheers!